packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/resolve_import_errors.ts

```diff
@@ -55,6 +55,11 @@ export interface ResolveSavedObjectsImportErrorsOptions {
   namespace?: string;
   /** If true, will create new copies of import objects, each with a random `id` and undefined `originId`. */
   createNewCopies: boolean;
+  /**
+   * If true, Kibana will apply various adjustments to the data that's being retried to import to maintain compatibility between
+   * different Kibana versions (e.g. generate legacy URL aliases for all imported objects that have to change IDs).
+   */
+  compatibilityMode?: boolean;
```

Comment:
@azasypkin I'm working in code related to import saved objects and went hunting down how `compatibilityMode` works in `resolve_import_errors`. The docs for this HTTP API need to be updated to include the new query option.

Reply:
Ah, good catch, thanks @TinaHeiligers! I'll update docs for this API as well.